pkg/automationconfig/automation_config.go

```diff
@@ -135,7 +135,8 @@ type Process struct {
 	ProcessType                 ProcessType  `json:"processType"`
 	Version                     string       `json:"version"`
 	AuthSchemaVersion           int          `json:"authSchemaVersion"`
-	LogRotate                   *AcLogRotate `json:"LogRotate,omitempty"`
```

Comment:
this does not look good, I am wondering whether the agent was able to read that correctly